modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+= APIkit for GraphQL Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit"]
+
+== Configurations
+---
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
+| API definition | String | Defines the path to the API definition resource, e.g.: resource::7ddfc767-c5b3-4583-9997-a468e251acd2:posts:1.0.0:graphql:zip:api.graphql |  | *x*
+| Introspection Enabled | Boolean | Enables schema introspection. | false | 
```

Comment:
Should false be init caps?

Reply:
GraphQL is a case-sensitive language, so "false" is not the same as "False", and that's why it says "false".
Thanks for asking!